docs/PAYJOIN.md

```diff
@@ -4,7 +4,6 @@
 reading this).
 
 This document does **not** discuss why PayJoin is interesting or the general concept.
-For that, see [this](https://joinmarket.me/blog/blog/payjoin/) post.
```

Comment:
This can be updated to https://reyify.com/blog/payjoin.

Reply:
perfect, thanks done
do you also know which video was linked?
(I tried finding it with a web archive explorer, but no success)

Reply:
No idea, maybe @AdamISZ can comment.

Reply:
I think that video is no longer up anywhere. Not sure it was so great anyway :) So best just delete the link, thanks for taking a look @MarnixCroes !